frontend/src/v5/ui/controls/inputs/passwordField/passwordField.component.tsx

```diff
@@ -15,9 +15,8 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { TextField } from '@controls/inputs/textField/textField.component';
-import styled from 'styled-components';
+import { TextField, TextFieldProps } from '@controls/inputs/textField/textField.component';
 
-export const PasswordField = styled(TextField).attrs({
-	type: 'password',
-})``;
+export const PasswordField = (props: TextFieldProps) => (
```

Comment:
should login.component.tsx use this, or is the styling too different to not make it worth it?